com.korbsstudio.penpot-deskop.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You need to explicitly install this file in `build-commands` to `/app/share/appdata/$ID.appdata.xml`. This is why the build fails.